docs/variants/msi_z790/releases.md

```diff
@@ -13,6 +13,112 @@ For details about our release process please read
 
 </center>
 
+## v0.9.1 - 2024-01-17
```

Comment:
@miczyg1 date needs update to today

Reply:
Changed: 2c3298eacb9deff6c99330f7b03746d654172a7d